bin/next-dev

```diff
@@ -5,9 +5,15 @@ import parseArgs from 'minimist'
 import { existsSync } from 'fs'
 import Server from '../server'
 import { printAndExit } from '../lib/utils'
+import pkg from '../../package.json'
+import updateNotifier from 'update-notifier'
 
 process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Is this check needed? (Doesn't `next-dev` start the server in development mode?)